career-link-webapi/Controllers/PostController.cs

```diff
@@ -13,16 +15,80 @@ public PostController(IPostService taskService)
             _postService = taskService;
         }
 
-        [HttpGet("allpost")]
```

Comment:
if you change the end point path, do you make sure the frontend using this endpoint still working ?